src/jade/tabs/aud.jade

```diff
@@ -0,0 +1,109 @@
+              p(l10n) .98% for deposits and withdrawals&#32;
+              p(l10n)
+                a(href="https://coinexgateway.com/Gateway/Fees.html", target="_blank") More details 
+        .col-md-3
+            .title(l10n) Rating
+            .description(l10n)
+              a(href="https://coinist.co/ripple/gateways/coinex", target="_blank") read reviews
+
+      .row.fund-tab-description
+        .col-xs-12.col-md-8.col-lg-6
+          .descriptor(ng-show="!audConnected", l10n) Ripple Trade has partnered with Ripple Australia to provide easier access to AUD. Following this action will enable Ripple Australia to hold AUD on your behalf.
+
+      .row.fund-tab-description
+        .col-xs-7.col-sm-5.col-md-4
+          button.btn.btn-large(ng-show="!audConnected && !loading", type="submit", ng-click="save_account()", ng-disabled="!account.Balance") Add AUD Coinex
+          button.btn.btn-large(ng-show="!audConnected && loading", type="submit", ng-disabled="loading") Adding...
+          button.btn.btn-large(ng-show="showInstructions && audConnected", type="submit", ng-click="toggle_instructions()", l10n) Hide instructions
+          button.btn.btn-large(ng-show="audConnected && !showInstructions", type="submit", ng-click="toggle_instructions()", l10n) Show instructions
+
+        .connect.col-xs-5.col-sm-7.col-md-8(ng-hide="loading")
+          .description(ng-show="!audConnected && account.Balance", l10n) 
+            i.fa.fa-times
+            |  Not connected
+          .description(ng-show="audConnected && account.Balance", l10n)
+            i.fa.fa-check
+            |  Connected
+          .description(ng-show="!account.Balance", l10n) Your account has to be activated before you can add a gateway account.
+
+      .row(ng-show="showInstructions && audConnected")
+        .instructions.col-md-10
+          a.dismiss(href="", id="hide", ng-click="toggle_instructions()", l10n)  ×
+          .title(l10n) To Initiate Australian Dollar Deposits and Withdrawals
+          .description(l10n) 
+            | Register or login to your Coinex account. You will need to submit proof of identity.&#32;
+            a(href="https://coinexgateway.com/Gateway/Verification.html", target="_blank") Details here.&#32;
+            
+          .title(l10n) Deposit Options
+          .description(l10n) 
+            | Coinex accepts deposits by the following methods:
+            ul 
+              li
+                p(l10n) International bank transfer
+              li
+                p(l10n) Ripple transaction
+              li
+                p(l10n) Stellar transaction
+
+          .title(l10n) Withdrawal Options
+          .description(l10n)
+            | Coinex accepts withdrawals by the following methods:
+            ul
+              li
+                p(l10n) Domestic or international bank transfer
+              li
+                p(l10n) Ripple transaction
+              li
+                p(l10n) Stellar transaction
```

Comment:
@annatonger Do we want to make mention of Stellar anywhere in Ripple Trade? :question: